posthog/api/user.py

```diff
@@ -59,6 +59,7 @@ class UserSerializer(serializers.ModelSerializer):
     is_impersonated = serializers.SerializerMethodField()
     is_2fa_enabled = serializers.SerializerMethodField()
     has_social_auth = serializers.SerializerMethodField()
+    postgres_id = serializers.SerializerMethodField()
```

Comment:
I don't think we should expose this. Instead we can just use the email and link to the list view like:

https://app.posthog.com/admin/posthog/user/?q=ben%40posthog.com

Reply:
hmm... this reminds me I need to also call out the current project name as the user might have changed their organization and project by this point.